Products/CMFPlone/controlpanel/browser/redirects-controlpanel.pt

```diff
@@ -0,0 +1,174 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      xml:lang="en" lang="en"
+      metal:use-macro="context/prefs_main_template/macros/master"
+      i18n:domain="plone">
+
+  <body>
+    <div metal:fill-slot="prefs_configlet_main">
+      <dl
+          tal:condition="view/errors"
+          class="portalMessage error">
+        <dt>Error</dt>
+        <dd>
+          <p i18n:translate="error_bulk_upload">
+            No aliases were added. Please correct these errors in your CSV file and try again:
```

Comment:
Please change the use of the word Alias. It's not obvious to users what this means. Users think of alternative urls or short urls as the words they use in my experience. Redirection might also be possible but it could be confusing with other forms of redirection